### ISSUE TITLE
Node flags not passed spawned process on win32

### ISSUE THREAD

**Issue by ProTip** [tj/commander.js#559]
This prevents harmony flags being passed into sub-commands.

``` js
var proc;
  if (process.platform !== 'win32') {
    if (isExplicitJS) {
      args.unshift(bin);
      // add executable arguments to spawn
      args = (process.execArgv || []).concat(args);

      proc = spawn('node', args, { stdio: 'inherit', customFds: [0, 1, 2] });
    } else {
      proc = spawn(bin, args, { stdio: 'inherit', customFds: [0, 1, 2] });
    }
  } else {
    args.unshift(bin);
    proc = spawn(process.execPath, args, { stdio: 'inherit'});
  }
```

**Comment by shadowspawn** [comment]
A new option might be using NODE_OPTIONS?
https://nodejs.org/api/cli.html#cli_node_options_options

This issue has not had any activity in over six months. It isn't likely to get acted on due to this report.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
 
Thank you  for your contributions.

**Comment by shadowspawn** [comment]
Hopefully this issue will be fixed by #611
